### DISCUSSION TITLE
有没有流量统计的功能或插件？

### DISCUSSION THREAD

**Discussion by UniIsland** [vernesong/OpenClash#2389] [General]
在软路由上使用 openclash，当前连接的面板没有历史流量的统计记录。
想问问大家有没有办法针对不同域名、策略组，做流量的分时段统计呢，有没有带这个功能的插件？

**Comment by InspoOnU** [comment]
Clash的原作者出过类似的软件Clash-Tracing
https://github.com/Dreamacro/clash-tracing
